modules/database/src/ioc/dbStatic/dbStaticLib.c

```diff
@@ -1478,6 +1478,42 @@ long dbDeleteAliases(DBENTRY *pdbentry)
     return 0;
 }
 
+static long dbDeleteRecordLinks(dbRecordType *rtyp, struct dbCommon *prec)
+{
+    short i;
+
+    for (i=0; i<rtyp->no_links; i++) {
+        dbFldDes *pflddes = rtyp->papFldDes[rtyp->link_ind[i]];
+        DBLINK *plink = (DBLINK *)(((char *)prec) + pflddes->offset);
+
+        // TODO: How to handle different link types?
```

Comment:
I think the answer is:

https://github.com/epics-base/epics-base/blob/72d50ce2749bbc791907a32c1ed77af1333a21f5/modules/database/src/ioc/dbStatic/dbStaticLib.c#L107

Reply:
Addressed in https://github.com/epics-base/epics-base/pull/505/commits/3dc1ef2b73509346d86d3d9e3b4940161f13622d.